### PR TITLE
feat(zipcloak): add zipcloak completion spec

### PR DESCRIPTION
**Pull request by sa1sr1**

### CODE REVIEW

**Comment by SawanSrivastava1210** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/zipcloak.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { filepaths } from "@fig/autocomplete-generators";
```
Comment (fedeci) [inline src/zipcloak.ts:1]: This is not required when using the `template: ` sugar.
```suggestion
```

File: src/zipcloak.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
+
+const completionSpec: Fig.Spec = {
+  name: "zipcloak",
+  description: "Encrypt entries in a zipfile",
+  args: {
+    name: "zipfile",
+    description: "Zipped file to encrypt entries into",
+    template: ["filepaths"],
```
Comment (fedeci) [inline src/zipcloak.ts:8]:
```suggestion
    template: ["filepaths"],
    suggestCurrentToken: true,
```

File: src/zipcloak.ts [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      description: "Quiet operation. Suppresses some informational messages",
+    },
+    {
+      name: ["--output-file", "-O"],
+      description:
+        "Write output to new archive zipfile, leaving original archive as is",
+      args: {
+        name: "zipfile",
+        description: "New zipfile that will be written to",
+        template: ["filepaths"],
```
Comment (fedeci) [inline src/zipcloak.ts:43]: Same suggestCurrentToken requirement here.

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks, I left a couple comments to fix.

**Comment by fedeci** [comment]
@sa1sr1 the CI is failing, can you run `npm run lint:fix` in your local repo clone and push the changes?

**Comment by grant0417** [comment]
Thanks for the contrib!

**Pull request merged**